Move sentence about address rewriting (#319)
quicwg/ops-drafts: draft-ietf-quic-manageability.md

variables like congestion control parameters will be reset protecting
the new path from overload.

Therefore use of address rewriting to ensure routing stablity as an approach

> to simplify existing operations can open QUIC up to various attacks as it 

Author: ```suggestion
to simplify existing operations can open QUIC up to various attacks as it
```

> Therefore use of address rewriting to ensure routing stablity as an approach
> to simplify existing operations can open QUIC up to various attacks as it
> conceals client address changes and as such masks important signals that
> drive security mechanisms.
> 

Reviewer: ```suggestion
Therefore, the use of address rewriting to ensure routing stability
can open QUIC up to various attacks, as it
conceals client address changes, and as such masks important signals that
drive security mechanisms.

```
